marklogic-unit-test-modules/src/main/ml-modules/root/test/test-helper.xqy

```diff
@@ -787,7 +846,7 @@ declare function test:assert-meets-maximum-threshold(
     test:success()
   else
     fn:error(
-      xs:QName("ASSERT-MEETS-MAXIMUM-THRESHOLD-FAILED"),
+      xs:QName("ASSERT-LESS-THAN-OR-EQUAL-FAILED"),
       fn:string-join((
         $message,
         "actual: " || xdmp:describe($actual) || " is greater than maximum: " || xdmp:describe($maximum)
```

Comment:
Mind changing the error message here to more closely align with the function name?

Instead of:

`"actual: " || xdmp:describe($actual) || " is greater than maximum: " || xdmp:describe($maximum)`

We should say:

`xdmp:describe($actual) || " is not less than or equal to  " || xdmp:describe($maximum)`

Reply:
Sounds good.  I'll make the same change for the GTE function too.